pyproject.toml

```diff
@@ -46,6 +47,11 @@ termcolor = "^1.1.0"
 twine = "^3.2.0"
 pre-commit = "^2.17.0"
 
+[tool.poetry.group.dev.dependencies]
+ropevim = "^0.8.1"
+pylint = "^2.15.0"
+pudb = "^2022.1.2"
```

Comment:
Is this some new way of defining dev dependencies?

Reply:
I'm new to poetry, but I got a warning that `poetry add --dev` was deprecated, and that one should use `poetry add --group=dev` from now on. So I just went with that.

Poetry version 1.2.0